internal/cli/tables-to-go-cli.go

```diff
@@ -273,21 +274,12 @@ func getNullType(settings *settings.Settings, primitive string, sql string) stri
 
 func toInitialisms(s string) string {
 	for _, substr := range initialisms {
-		idx := indexCaseInsensitive(s, substr)
-		if idx == -1 {
-			continue
-		}
-		toReplace := s[idx : idx+len(substr)]
-		s = strings.ReplaceAll(s, toReplace, substr)
+		m := regexp.MustCompile(`(?i)` + substr)
+		s = string(m.ReplaceAll([]byte(s), []byte(substr)))
```

Comment:
Usually seeing a regex in a loop is a huge performance penalty - but since this is not a real-time API and "just" a CLI tool, lets leave it for now, we can improve anytime if we encounter real performance issues.

One change request tho, lets use `s = m.ReplaceAllString(s, substr)` - this way we can get rid of all the back and forth casting between bytes and strings.